google.js

```diff
@@ -64,8 +63,21 @@ function processArticle(article) {
 
 window.onload = function() {
 
-  //chrome.storage.local.set({'filters' : ["trump", "clinton"]});
-  chrome.storage.local.get('filters', function(items){ if (typeof items != undefined) {filterWords = items.filters;} parsePage() });
+  //chrome.storage.local.set({'filterWords' : ["trump", "clinton"]});
+  //chrome.storage.local.get('filterWords', function(items){ if (typeof items != undefined) {filterWords = items.filterWords;} parsePage() });
+  chrome.storage.local.get('filterWords',
+    function(items) {
+      if (typeof items != 'undefined' && typeof items.filterWords != 'undefined') {
+        filterWords = items.filterWords;
+debug(items);
```

Comment:
It seems like you're purposefully not indenting these `debug` lines, but you did indent the debug call on line 2 of `isFilteredWord`.  My preference is to keep the indentation consistent.